clients/lnurl.js

```diff
@@ -1,12 +1,31 @@
 
-const { requestInvoice } = require('lnurl-pay')
+const { requestInvoice, utils } = require('lnurl-pay')
+const { isOnionUrl, decodeUrlOrAddress } = utils
+const tor_axios = require('tor-axios');
+const config = require('../config.json')
+
+const tor = tor_axios.torSetup({
+    ip: 'localhost',
```

Comment:
Can we make `localhost` configurable? In Umbrel, tor is never running on localhost...

Reply:
updated